### ISSUE TITLE
We show "Price :" for an artwork that is considered not for sale

### ISSUE THREAD

**Issue by orta** [artsy/eigen#865]
<img width="423" alt="screen shot 2015-10-08 at 12 28 59" src="https://cloud.githubusercontent.com/assets/49038/10364057/c04eed78-6db8-11e5-8f91-1c05f9478630.png">

<img width="1149" alt="screen shot 2015-10-08 at 12 32 37" src="https://cloud.githubusercontent.com/assets/49038/10364064/c22fab78-6db8-11e5-8b0e-f68e97907b61.png">

**Comment by alloy** [comment]
From https://github.com/artsy/eigen/blob/df3348a8b76b1dce3b95e847f1def1b0f028d119/Artsy/Views/Artwork/ARArtworkActionsView.m#L308-L313:

```
- (BOOL)showPriceLabel
{
    // Check if a price is set OR the price should be hidden - this way we can show the "Contact For Price"
    // message whether or not a price has been entered
    return (self.artwork.price.length || self.artwork.isPriceHidden.boolValue) && !self.artwork.hasMultipleEditions && (self.artwork.inquireable.boolValue || self.artwork.sold.boolValue);
}

- (BOOL)showPriceLabel
{
    return (priceNotEmpty || isPriceHidden) && !hasMultipleEditions && (inquireable || sold);
}

(lldb) p (BOOL)self.artwork.price.length
(BOOL) $18 = NO
(lldb) p (BOOL)self.artwork.isPriceHidden.boolValue
(BOOL) $19 = YES
(lldb) p (BOOL)self.artwork.hasMultipleEditions
(BOOL) $20 = NO
(lldb) p (BOOL)self.artwork.inquireable.boolValue
(BOOL) $21 = YES
(lldb) p (BOOL)self.artwork.sold.boolValue
(BOOL) $22 = NO

- (BOOL)showPriceLabel
{
    return (NO || YES) && !NO && (YES || NO);
}

- (BOOL)showPriceLabel
{
    return (NO || YES) && YES && (YES || NO);
}

- (BOOL)showPriceLabel
{
    return YES && YES && YES;
}
```

I believe the `isPriceHidden` value is handled incorrectly. The price should be hidden, so the first bool should end-up being `NO`, not `YES`.

**Comment by orta** [comment]
>  The price should be hidden, so the first bool should end-up being NO, not YES.

Why? The "price" is hidden, given that it's classed as not for sale on CMS.

**Comment by alloy** [comment]
It ends up adding the view here https://github.com/artsy/eigen/blob/df3348a8b76b1dce3b95e847f1def1b0f028d119/Artsy/Views/Artwork/ARArtworkActionsView.m#L136-L140

**Comment by alloy** [comment]
I get what you’re saying now, indeed no price is being shown.

**Comment by alloy** [comment]
Fixed by #867.
